### ISSUE TITLE
Buggy editors availability check

### ISSUE THREAD

**Issue by pierreozoux** [ONLYOFFICE/onlyoffice-nextcloud#880]
We noticed that oo was deactivated after the update.

Going to the server admin and clicking save to the oo configuration tab fixed the issue.

To fix it in bash, we checked and noticed the following:
```
/usr/local/bin/php occ config:app:get onlyoffice settings_error --default-value=novalue 2>/dev/null' | grep SSL
SSL CA bundle not found: /tmp/oc_tmp_xxxx-.crt
```

and this command fixed the issue:
```
php occ config:app:set onlyoffice settings_error --value ''
```

I don't know what happened, but I wanted to document it, in case other might find this info usefull!

Cheers

**Comment by KornKalle** [comment]
+1 We are experiencing the same thing. Thanks for this occ command. I was searching for ages for documentation about the commands available with the connector app

**Comment by pierreozoux** [comment]
This is strange, I have to do it again :/ Wondering what is happening.

**Comment by pierreozoux** [comment]
It seems that this is due to this commit:
https://github.com/ONLYOFFICE/onlyoffice-nextcloud/commit/03251b8d24e6f004abd117f092931ac695ff0821

Once in a while, the app is disabled.

It is currently impossible to disable the cron.
We changed the regularity to test only every 30d and not every 24h.

So either the check is not good, or our server is unstable.

**Comment by pierreozoux** [comment]
And this is the Pull request:
[Feature/editors availability check](https://github.com/ONLYOFFICE/onlyoffice-nextcloud/pull/795)

**Comment by pierreozoux** [comment]
Actually, we can disable the check with `editors_check_interval` set to 0.

**Comment by pierreozoux** [comment]
Ok, we have to wait this PR:
https://github.com/ONLYOFFICE/onlyoffice-nextcloud/pull/874/files

:)

**Comment by SergeyKorneyev** [comment]
There's another fix coming that should help with this issue: https://github.com/ONLYOFFICE/onlyoffice-nextcloud/issues/864#issuecomment-1743257243

**Comment by SergeyKorneyev** [comment]
The version that includes the fixes (7.9.2/8.2.2) has just been released. Please try updating the connector and check if that helps.
